styles.css

```diff
@@ -0,0 +1,126 @@
+body {
+  margin: 0;
+}
+
+.wrapper {
+  margin: auto;
+  width: 80%;
+}
+
+.main-content__title {
+  font-family: 'PT Serif', serif;
+}
+
+.logo-name {
+  font-family: 'PT Serif', serif;
+}
+
+.main-content__description {
+  font-family: 'Roboto', sans-serif;
+}
+
+.menu {
+  font-family: 'Roboto', sans-serif;
+  padding: 47px 0;
+}
+
+.main-content__button {
+  font-family: 'Roboto', sans-serif;
+  padding: 20px 10px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6252582)